lua/lspconfig/server_configurations/dartls.lua

```diff
@@ -1,8 +1,15 @@
 local util = require 'lspconfig.util'
 
+local bin_name = 'dart'
+local cmd = { bin_name, 'language-server', '--protocol=lsp' }
+
+if vim.fn.has 'win32' == 1 then
+  cmd = { 'cmd.exe', '/C', bin_name, 'language-server', '--protocol=lsp' }
+end
```

Comment:
ternary is more readable, avoids nesting, etc.

```suggestion
local cmd = (vim.fn.has 'win32' == 1
    and { 'cmd.exe', '/C', bin_name, 'language-server', '--protocol=lsp' }
    or { bin_name, 'language-server', '--protocol=lsp' })
```

Reply:
Thanks, I didn't know about the lua ternary.